src/pages/layout/css-utilities.md

```diff
@@ -176,6 +176,44 @@ The table below shows the default behavior, where `{modifier}` is any of the fol
 | `float-lg-{modifier}`  | Applies the modifier to the element when `min-width: 992px`.        |
 | `float-xl-{modifier}`  | Applies the modifier to the element when `min-width: 1200px`.       |
 
+## Element Visibility
+
+The display CSS property determines if an element should be visible or not. The element will still be in the DOM, but not rendered, if it is hidden.
+
+```html
+<ion-grid>
+  <ion-row>
+    <ion-col hidden>
+      <div>
+        <h3>hidden</h3>
+        You can't see me.
+      </div>
+    </ion-col>
+    <ion-col>
+      <div>
+        <h3>not-hidden</h3>
+        You can see me!
+      </div>
+    </ion-col>
+  </ion-row>
+</ion-grid>
+```
+
+| Attribute     | Style Rule                      | Description                                                                                               |
+|---------------|---------------------------------|-----------------------------------------------------------------------------------------------------------|
+| `hidden`      | `display: hidden`               | The element will be hidden. 
+
+### Responsive Visibility Attributes
+
+There are also additional attributes to modify the visibility based on the screen size. Instead of just `hidden` as a sole attribute, use `hidden-{breakpoint}-{dir}` to only use the attribute on specific screen sizes, where `{breakpoint}` is one of the defined breakpoints in the `--screen-breakpoints` variable, and `{dir}` is whether the element should be hidden on all screen sizes above or below the current one.
```

Comment:
I can't seem to find any reference to these classes in the Ionic styles. If we had them at some point, we don't seem to have them anymore. @brandyscarney, know anything about these?

Reply:
It depends on the linked PR being merged: ionic-team/ionic#17359

I asked him to add them in Ionic & document them here. 🙂

Reply:
Yeah, the linked PR would have the actual meat, and I updated the docs in tandem and submitted the PR for them both around the same time.